.github/workflows/contracts.yml

```diff
@@ -32,3 +32,13 @@ jobs:
     - uses: actions/checkout@v3
     - uses: ./.github/actions/setup-contracts-clients
     - run: yarn test
+
+  # ensure gas measurement script runs
+  test-gas-measurements:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: ./.github/actions/setup-contracts-clients
+    - uses: ./.github/actions/local-contract-deploy
+    - run: yarn hardhat run ./scripts/measure_gas/run.ts --network gethDev
```

Comment:
The results of this get written to a file which we can't see after CI runs. We should upload it as an artifact or add a `--stdout` option so that we can see the measurement results on CI.

Could be done as followup.

Reply:
This is mainly here as a smoke test to let someone know (such as myself when developing this) if their changes break this script. See https://github.com/web3well/bls-wallet/issues/140 for work which will upload the results.